documentation/Using-OVS-DPDK-and-COR.md

```diff
@@ -0,0 +1,97 @@
+
+## Installation of DPDK and OVS on host system
+
+There is tight coupling between OVS and DPDK versioning, and what is available via distributions is often very out of date.  The directions below will instruct you how to build and configure from source. You MUST update the packages listed regularly to ensure your Clear Containers benefit from performance improvements, new features and most importantly CVE security fixes.
+
+As a prerequisite to building, you'll need a few packages on your host.  As an example, on an Ubuntu host system you'd need to install as follows:
+```
+$ sudo apt-get -y install autoconf automake kernel-common \
+    libpcap-dev libtool python python-six
+```
+
+Next we will set up DPDK:
+```
+$ git clone http://dpdk.org/git/dpdk -b v16.11 $HOME/dpdk
+$ cd $HOME/dpdk
+$ export RTE_SDK=$(pwd)
+$ export RTE_TARGET=x86_64-native-linuxapp-gcc
+$ make -j install T=$RTE_TARGET DESTDIR=install EXTRA_CFLAGS='-g'
+$ sudo cp -f install/lib/lib* /lib64/
+$ export DPDK_BUILD_DIR=x86_64-native-linuxapp-gcc
+```
+Once this completes with out issue, build and configure OVS:
+```
+$ git clone https://github.com/openvswitch/ovs -b branch-2.6 $HOME/ovs
+$ cd $HOME/ovs
+$ ./boot.sh
+$ ./configure --prefix=/usr --sysconfdir=/etc --localstatedir=/var \
+    --with-dpdk="$RTE_SDK/$DPDK_BUILD_DIR"  --disable-ssl --with-debug  CFLAGS='-g'
+$ sudo -E make install -j
+```
+
+## Running OVS-DPDK on host system
+Once installed without error, we'll need to start up the ovsdb-server and ovs-vswitchd. The Open_vSwitch configuration settings will vary depending on the hardware you are running on (ex: lcore-mask, socket-mem). An example configuration is shown below, which should be adjusted for your setup:
+
+```
+$ sudo mkdir -p /var/run/openvswitch
+$ sudo killall ovsdb-server ovs-vswitchd
+$ rm -f /var/run/openvswitch/vhost-user*
+$ sudo rm -f /etc/openvswitch/conf.db
+
+$ export DB_SOCK=/var/run/openvswitch/db.sock
+$ sudo -E ovsdb-tool create /etc/openvswitch/conf.db /usr/share/openvswitch/vswitch.ovsschema
+$ sudo -E ovsdb-server --remote=punix:$DB_SOCK --remote=db:Open_vSwitch,Open_vSwitch,manager_options --pidfile --detach
+
+$ sudo -E ovs-vsctl --no-wait init
+$ sudo -E ovs-vsctl --no-wait set Open_vSwitch . other_config:dpdk-lcore-mask=0x3
+$ sudo -E ovs-vsctl --no-wait set Open_vSwitch . other_config:dpdk-socket-mem=1024,0
+$ sudo -E ovs-vsctl --no-wait set Open_vSwitch . other_config:dpdk-init=true
+
+$ sudo sysctl -w vm.nr_hugepages=4096
+$ sudo mount -t hugetlbfs -o pagesize=2048k none /dev/hugepages
+
+$ sudo -E ovs-vswitchd unix:$DB_SOCK --pidfile --detach --log-file=/var/log/openvswitch/ovs-vswitchd.log
+$ ps -ae | grep ovs
```

Comment:
maybe we should add an indication of what processes we should see listed here? I